scripts/gui/main_menu.mjs

```diff
@@ -227,6 +246,19 @@ function newGame(){
     }
 }
 
+function enterFreebuildEditor(){
+    if(common.isInEngine()){
+        Thrive.start();
+        switchToMicrobeHUD();
+        jams.Pause();
+        $("#toolsMenu").slideUp("fast", () => {
+            $("#mainMenu").slideDown("fast");
+        });
+        document.getElementById("topLevelMenuContainer").style.display = "none";
```

Comment:
Why is this here? the normal route doesn't call this?

Reply:
Lines 254 to 256 is to reset the menu back to the main page when returning back to it from in game. If I remove the code the game will return to the tools submenu. Although I probably don't need the animation here.

You're right about line 257, I'll remove it.